src/sentry/api/authentication.py

```diff
@@ -380,17 +410,18 @@ def authenticate_token(self, request: Request, token_str: str) -> tuple[Any, Any
         token_hashed = hash_token(token_str)
 
         if SiloMode.get_current_mode() == SiloMode.REGION:
-            token = OrgAuthTokenReplica.objects.filter(
-                token_hashed=token_hashed,
-                date_deactivated__isnull=True,
-            ).last()
-            if token is None:
+            try:
+                token = OrgAuthTokenReplica.objects.get(
+                    token_hashed=token_hashed,
```

Comment:
I'm not seeing an index on this field. I'll get an index added so that we don't have a repeat of the table scan.

Reply:
https://github.com/getsentry/sentry/pull/66063